# robot loaded or unloaded succesfully
bool loaded

# package
Package package
